src/Files.App/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -105,15 +105,15 @@
 				</Grid>
 
 				<!--  Options  -->
-				<!-- Use Visibility because x:Load does not reflect the SelectedIndex value in the combo box -->
+				<!--  Use Visibility because x:Load does not reflect the SelectedIndex value in the combo box  -->
 				<ComboBox
 					x:Name="ConflictOptions"
 					Grid.Column="2"
 					Width="200"
 					HorizontalAlignment="Right"
 					VerticalAlignment="Center"
-					Visibility="{x:Bind IsConflict, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}"
-					SelectedIndex="{x:Bind ConflictResolveOption, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay, Converter={StaticResource ConflictResolveOptionToIndexConverter}}">
+					SelectedIndex="{x:Bind ConflictResolveOption, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay, Converter={StaticResource ConflictResolveOptionToIndexConverter}}"
+					Visibility="{x:Bind IsConflict, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}">
```

Comment:
These changes look unrelated, let's try to limit PRs to their purpose. Makes it easier to review and to look back later on.